### PR TITLE
:recycle: Connecting Compose ProfileCardScreen as an iOS app screen

### LINKED ISSUE #814: Connecting Compose ProfileCardScreen as an iOS app screen
Currently, the iOS app only has a native screen for creating a Profile card. However, instead of using this native implementation, we want to integrate the Compose Multiplatform ProfileCardScreen into the iOS app. The goal is to have the Compose editing screen appear when the profile card icon in the bottom navigation bar is tapped in the iOS app.

### PR DESCRIPTION
**Pull request by Corvus400**
## Issue
- close #814

## Overview (Required)
- The parts of the UI that were implemented in SwiftUI have been changed to show the UI created in Jetpack Compose.
- Sharing functionality should be outside the scope of this task, so we have not implemented it.

## Movie (Optional)
Before | After
:--: | :--:
<video src="https://github.com/user-attachments/assets/d83db853-8d8a-4842-bcb9-cdf3b4a2f182" width="300" > |

### CODE REVIEW

File: feature/profilecard/src/iosMain/kotlin/io/github/droidkaigi/confsched/profilecard/ImageBitmapToUiImage.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        height = height.toULong(),
+        bitsPerComponent = 8u,
+        bytesPerRow = (4 * width).toULong(),
+        space = colorSpace,
+        bitmapInfo = bitmapInfo,
+    )
+
+    val cgImage = CGBitmapContextCreateImage(context)
+    return cgImage?.let { UIImage.imageWithCGImage(it) } ?: UIImage()
+}
```
Comment (Corvus400) [inline feature/profilecard/src/iosMain/kotlin/io/github/droidkaigi/confsched/profilecard/ImageBitmapToUiImage.kt:35]: The exact same implementation is available in another module.
However, I was not sure where to put this process, so I copied it and put it as is.
Which module should I put it in? 🤔

File: app-ios/Sources/App/RootView.swift [hunk 2]
```diff
@@ -189,14 +189,9 @@ public struct RootView: View {
...
-                    store: Store(
-                        initialState: .init(),
-                        reducer: {
-                            ProfileCardInputReducer()
-                        }
-                    )
-                )
+                KmpProfileCardComposeViewControllerWrapper { text, image in
+                    // TODO Implement a sharing function.
+                }
```
Comment (Corvus400) [inline app-ios/Sources/App/RootView.swift:194]: > The goal is to have the Compose editing screen appear when the profile card icon in the bottom navigation bar is tapped in the iOS app.

Since the goal of this task is the above, the share function is not part of this PR.

File: feature/profilecard/src/iosMain/kotlin/io/github/droidkaigi/confsched/profilecard/ImageBitmapToUiImage.kt (hunk 1 above)
Comment (takahirom) [reply feature/profilecard/src/iosMain/kotlin/io/github/droidkaigi/confsched/profilecard/ImageBitmapToUiImage.kt:35]: How about putting this to core/ui?👀

Comment (takahirom) [reply feature/profilecard/src/iosMain/kotlin/io/github/droidkaigi/confsched/profilecard/ImageBitmapToUiImage.kt:35]: We can have another pull request for this👀

**Review by takahirom — APPROVED** [review]
Looks good to me!
